File: app/src/main/java/com/hujiayucc/qqsender/utils/Toast.java
package com.hujiayucc.qqsender.utils;

import android.annotation.SuppressLint;
import com.hujiayucc.qqsender.Applications;

public class Toast {
    private static final int LONG = 1;
    private static final int SHORT = 0;

    @SuppressLint("WrongConstant")
    public static void Long(String message) {
        android.widget.Toast.makeText(Applications.Companion.getContext(), message, LONG).show();
    }

    @SuppressLint("WrongConstant")
    public static void Short(String message) {
        android.widget.Toast.makeText(Applications.Companion.getContext(), message, SHORT).show();
    }
}

File: app/src/main/java/com/hujiayucc/qqsender/utils/Toast.kt
package com.hujiayucc.qqsender.utils

import android.annotation.SuppressLint
import android.widget.Toast
import com.hujiayucc.qqsender.Applications.Companion.context

object Toast {
    private const val LONG = 1
    private const val SHORT = 0

    @SuppressLint("WrongConstant")
    fun Long(message: String?) {
        Toast.makeText(context, message, LONG).show()
    }

    @SuppressLint("WrongConstant")
    fun Short(message: String?) {
        Toast.makeText(context, message, SHORT).show()
    }
}